docker-compose.yml

```diff
@@ -1,18 +1,33 @@
 version: '3.4'
 
+x-logging:
```

Comment:
From a git history / commit readability point of view, it would have been better to split this in a separate commit I think, since it seems unrelated to file service integration. At least that's what I would have done anyway :smile:.

Reply:
for small changes like this, I don't mind having a single commit. basically to keep consistent with was done in loket;

Reply:
Agree to disagree I guess :smile: